app/assets/stylesheets/hera/modules/_textile.scss

```
@@ -172,68 +172,62 @@
     background: transparent;
     padding: 0;
 
+    .editor-field {
+      .textile {
+        border: 0;
+        border-right: 1px solid $border-color;
+        border-radius: 0;
+        box-shadow: none !important;
+        padding: 0.5rem 1.5rem 0.5rem 1rem;
+      }
+    }
+
     .textile-form {
       .add-field {
         margin: 0.4rem 0.75rem;
       }
 
       .textile-form-body {
-        background-color: $secondary-bg;
-        border: 1px solid $border-color;
-        border-radius: 5px;
+        background-color: $white;
+        border-right: 1px solid $border-color;
       }
 
       .textile-form-field {
-        border-bottom: 1px solid transparent;
-        border-top: 1px solid transparent;
-        margin-bottom: -1px;
+        border-left: 4px solid transparent;
+        margin: 0 1.5rem 2.5rem 0;
+        padding-left: 0.5rem;
         position: relative;
 
         .actions {
-          bottom: 0;
+          bottom: -2.2rem;
           margin: 0 1rem;
           opacity: 0;
           padding: 0.5rem 0;
           position: absolute;
-          right: 0.5rem;
+          right: 0;
           transition: opacity 0.2s ease-in-out;
         }
 
         .editor-field {
-          margin: 0 1.5rem;
+          margin: 0;
 
           &.sticky-toolbar .editor-toolbar {
-            margin: 0 -1.5rem -1px -1.5rem;
             width: initial;
           }
         }
 
-        &:first-of-type:hover {
-          border-top-color: transparent;
-        }
-
         &:hover {
-          background-color: darken($secondary-bg, 2%);
-          border-bottom-color: $border-color;
-          border-top-color: $border-color;
+          border-left: 4px solid $light;
```

you're already setting this on L196, here you just need to update the color:
```suggestion
          border-color: $light;
```